src/bloom.cpp

```diff
@@ -48,11 +48,11 @@ CBloomFilter::CBloomFilter(const unsigned int nElements, const double nFPRate, c
 }
 
 // Private constructor used by CRollingBloomFilter
-CBloomFilter::CBloomFilter(const unsigned int nElements, const double nFPRate, const unsigned int nTweakIn) :
-    vData((unsigned int)(-1  / LN2SQUARED * nElements * log(nFPRate)) / 8),
+CBloomFilter::CBloomFilter(const size_t nElements, const double nFPRate, const uint32_t nTweakIn) :
+    vData((size_t)(-1  / LN2SQUARED * nElements * log(nFPRate)) / 8),
```

Comment:
This could be `static_cast<size_t>(...)` or simply `size_t(...)`.

Reply:
I wasn't sure I want to touch things like this. Because, well, if you start - you will then rewrite half of the bloom filter =) So I only replaced types. 

Still want me to adjust this stuff?

Reply:
I'm not saying to rewrite all the bloomfilter, but why no to change lines you're touching anyway?

It's nit anyway, up to you.

Reply:
Fixed